gradoop-flink/src/main/java/org/gradoop/flink/io/impl/text/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Contains classes related to the input and output of data in the gdl format.
+ */
+package org.gradoop.flink.io.impl.text;
```

Comment:
Just a minor comment: It only provides classes related to the output, right? Also wouldn't `gdl` instead of `text` be a better name for the package? (The javadoc mentions the format, so it should okay.)